src/clib/lib/config/config_parser.cpp

```diff
@@ -565,31 +574,54 @@ config_parse__(config_parser_type *config, config_content_type *content,
 
             // Add define
             else if (define_kw && (strcmp(define_kw, kw) == 0)) {
-                if (active_tokens < 3)
-                    util_abort("%s: keyword:%s must have exactly one (or more) "
-                               "arguments. \n",
-                               __func__, define_kw);
-
-                char *key = (char *)util_alloc_string_copy(
-                    stringlist_iget(token_list, 1));
-                char *value = stringlist_alloc_joined_substring(
-                    token_list, 2, active_tokens, " ");
+                if (active_tokens < 3) {
+                    content->parse_errors.push_back(fmt::format(
+                        "Keyword:{} must have two or more arguments.",
```

Comment:
Feedback is now correct? I take it that active_tokens 3 implies two arguments?

Reply:
Yes :)